packages/protocol/contracts/dapis/rrpDapi.sol

```diff
@@ -2,13 +2,91 @@
 pragma solidity 0.8.4;
 
 import "./reducers/Reducer.sol";
+import "./rrpDapiServer.sol";
 
 // A stub for an RRP dAPI contract
 // When the user wants to deploy a dAPI contract, they inherit the specific
 // reducer contract such as:
 //       contract myRrpDapi is rrpDapi, Median {
 //           // Any specific implementation required
 //       }
-abstract contract rrpDapi is Reducer {
-    // TODO: Implement
+abstract contract RrpDapi is Reducer {
```

Comment:
I initially created this `RrpDapi` contract, then implemented `RrpDapiServer` instead. We don't need a proxy contract to store the reduced answers in storage because the user may not need that. If they do, they implement this in their contract. This overwriting scheme should be implemented at `RrpDapiServer` to overwrite individual responses, as that is the main source of storage gas costs, not the reduced response.